test/pg/postgres_test.exs

```diff
@@ -15,7 +15,7 @@ defmodule EctoEnum.PostgresTest do
 
   alias Ecto.Integration.TestRepo
 
-  test "accepts int, atom and string on save" do
+  test "accepts atom and string on save" do
     user = TestRepo.insert!(%User{status: :registered})
```

Comment:
@ericmj after upgrading to Ecto 2.0. This test fails at this line with the following error:

```
 ** (ArgumentError) no extension found for oid `376414`
     stacktrace:
       (postgrex) lib/postgrex/types.ex:276: Postgrex.Types.fetch!/2
       (postgrex) lib/postgrex/types.ex:191: Postgrex.Types.encoder/2
       (elixir) lib/enum.ex:1184: Enum."-map/2-lists^map/1-0-"/2
       (postgrex) lib/postgrex/query.ex:101: DBConnection.Query.Postgrex.Query.encoders/2
       (postgrex) lib/postgrex/query.ex:48: DBConnection.Query.Postgrex.Query.describe/2
       (db_connection) lib/db_connection.ex:858: DBConnection.describe_execute/4
       (db_connection) lib/db_connection.ex:930: anonymous fn/4 in DBConnection.run_meter/5
       (db_connection) lib/db_connection.ex:987: DBConnection.run_begin/3
       (db_connection) lib/db_connection.ex:463: DBConnection.prepare_execute/4
       (ecto) lib/ecto/adapters/postgres/connection.ex:82: Ecto.Adapters.Postgres.Connection.execute/4
       (ecto) lib/ecto/adapters/sql.ex:228: Ecto.Adapters.SQL.sql_call/6
       (ecto) lib/ecto/adapters/sql.ex:426: Ecto.Adapters.SQL.struct/6
       (ecto) lib/ecto/repo/schema.ex:391: Ecto.Repo.Schema.apply/4
       (ecto) lib/ecto/repo/schema.ex:187: anonymous fn/11 in Ecto.Repo.Schema.do_insert/4
       (ecto) lib/ecto/repo/schema.ex:118: Ecto.Repo.Schema.insert!/4
       test/pg/postgres_test.exs:19: (test)
```

The `test/pg/test_helper.exs` already stops and starts the TestRepo. Any idea how to fix this?